packages/instance-model/lib/model.js

```diff
@@ -409,11 +396,15 @@ const InstanceModel = AmpersandModel.extend(
       return coll;
     },
 
+    shouldFetchDbAndCollStats() {
```

Comment:
By making these methods we don't have to be concerned about the event handlers in the store catching the event first and then calling refresh before they got updated here.